docker-compose.yml

```diff
@@ -47,3 +49,11 @@ services:
   airflow_scheduler:
     <<: *airflow
     command: scheduler
+
+  airflow_worker:
+    <<: *airflow
+    command: worker
+    ports:
+      - 8793:8793
```

Comment:
what's the port for?

Reply:
This port is used by the web server to request logs from workers.

Reply:
so it may not be necessary to expose it to the host, unless something outside of the containers uses it?

Reply:
will try without exposing the port.

Reply:
doesn't need to be exposed. Removing.